fireant/tests/widgets/test_reacttable.py

```diff
@@ -804,7 +810,6 @@ def test_dimx2_date_str_totals_all(self):
                         "$political_party": {
                             "display": "Totals",
                             "raw": "$totals",
-                            "hyperlink": "http://example.com/~~totals",
```

Comment:
I assume this is getting removed due to the added condition `if not is_totals`

why do we want to remove it? do hyperlinks not make sense for totals? (I have no context, if there's not much reasoning here that's fine. but if there is I'd be happy to hear it)

Reply:
~~totals is a constant marker we use for totals in the dataframe. It doesn't make sense to create a hyperlink for totals as that no id or name.